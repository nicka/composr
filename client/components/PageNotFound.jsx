import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () =>
  <div>
    <h1>Page not found</h1>
    <p>
      <Link replace to="/">Back to announcement</Link>
    </p>
  </div>;

export default PageNotFound;
