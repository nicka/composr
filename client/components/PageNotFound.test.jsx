import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import PageNotFound from './PageNotFound';

describe('PageNotFound', () => {
  it('renders the component', () => {
    const component = renderer.create(
      <MemoryRouter>
        <PageNotFound />
      </MemoryRouter>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
