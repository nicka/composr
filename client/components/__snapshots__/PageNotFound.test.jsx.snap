// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders the component 1`] = `
<div>
  <h1>
    Page not found
  </h1>
  <p>
    <a
      href="/"
      onClick={[Function]}
    >
      Back to announcement
    </a>
  </p>
</div>
`;
